src/pudl/constants.py

```diff
@@ -2641,6 +2644,13 @@
         'liquefied_natural_gas_storage': pd.BooleanDtype(),
         'longitude': float,
         'mercury_content_ppm': float,
+        'merge_address': pd.StringDtype(),  # Added by AES for Mergers table
+        'merge_city': pd.StringDtype(),  # Added by AES for Mergers table
+        'merge_company': pd.StringDtype(),  # Added by AES for Mergers table
+        'merge_date': 'datetime64[ns]',  # Added by AES for Mergers table
+        'merge_state': pd.StringDtype(),  # Added by AES for Mergers table
+        'merge_zip_4': pd.Int64Dtype(),  # Added by AES for Mergers table
+        'merge_zip_5': pd.Int64Dtype(),  # Added by AES for Mergers tables
```

Comment:
On the prefix here, are the addresses associated with a utility / operator / owner?  I think the other instances of address / city etc. are all for those corporate entities too, in which case it might not make sense to prefix here.  Fuzzy.

Separately, the Zip Code components need to be strings, since the leading zeroes are meaningful, and they'll get dropped if the values are encoded as numbers. We need a function to zero-pad these values though, since unfortunately they are stored as numbers in the EIA spreadsheets.  This function will also be useful for fixing the zip codes where they're read in for other tables.

Reply:
good to know

Reply:
I'll check re: the addresses

Reply:
So I've been poking around, and it seems like the `utility_id_eia` field corresponds to the `utility_name_eia` of the OLD (pre-merge) utility. As a result, the fields like `merge_address` cannot be mapped onto the ID from another table. This leads me to believe that we should keep the prefixes, what do you think?

Reply:
I don't think I understand well enough to have an opinion.